packages/bot-api/src/MessageHandler.ts

```diff
@@ -78,9 +78,9 @@ export abstract class MessageHandler {
     return this.account!.service!.user.getUsers(userIds);
   }
 
-  async removeUser(conversationId: string, userId: string): Promise<void> {
+  async removeUser(conversationId: QualifiedId, userId: QualifiedId): Promise<void> {
```

Comment:
I'm not sure where this function is being used and whether it's not a breaking change 🤔

Reply:
expect it to be a breaking change

Reply:
//will release as breaking change